LICENSE

```diff
@@ -1,6 +1,6 @@
 MIT License
 
-Copyright (c) 2021 Vyxal Organisation
+Copyright (c) 2022 Vyxal Organisation
```

Comment:
Wait I thought it was `(c) 2021-2022`?

Reply:
I'm pretty sure it should just be the current year